### DISCUSSION TITLE
lockscreen wallpaper cannot be changed

### DISCUSSION THREAD

**Discussion by FaridKaveh** [regolith-linux/regolith-desktop#1081] [Q&A]
Hi all, I'm using regolith 3 on Ubuntu 24.04. 

I set my lockscreen wallpaper a while ago in `~/.config/regolith3/.Xresources`.  Now I want to change it to match my home wallpaper. For my home wallpaper I just right-click on a photo and select "set as wallpaper" and that works. However, no matter what I do, I cannot change my lockscreen wallpaper. What I have tried: 

- Changing the `regolith.lockscreen.wallpaper.file` field in `.Xresources`
- Setting the `regolith.lockscreen.wallpaper.file` field to empty 
- Deleting the `.Xresources` file 
- Copying the repository given [here](https://github.com/orgs/regolith-linux/discussions/1073) to my `/etc/apt/sources.list` then running `sudo apt-get update && upgrade`

None of the above has worked. Any ideas?

**Comment by peterrus** [comment]
Are you on the Wayland (Sway) session? This (`regolith.lockscreen.wallpaper.file` in Xresources) has stopped working for me once I switched away from `i3`.
